build.gradle

```diff
@@ -139,8 +139,11 @@ dependencies {
     compile 'org.apache.httpcomponents:httpclient-cache:4.5.4'
     compile 'org.apache.logging.log4j:log4j-api:2.10.0' // the API of log4j 2
     compile 'org.apache.shiro:shiro-core:1.4.0'
-    compile 'org.apache.tika:tika-core:1.24.1'
-    compile 'org.apache.tika:tika-parsers:1.24.1'
+    compile 'org.apache.tika:tika-core:1.20'
```

Comment:
Hi Girish, 
We need to update tika-core to 1.24.1
https://issues.apache.org/jira/browse/OFBIZ-12080